content/docs/studio/index.md

```diff
@@ -7,16 +7,18 @@ tools [DVC](https://dvc.org/), [CML](https://cml.dev), [MLEM](https://mlem.ai/),
 enables you to seamlessly manage data and machine learning models, run and track
 experiments, and visualize and share results.
 
-<cards>
+https://www.youtube.com/watch?v=hKf4twg832g
```

Comment:
If we move the video to this page, we should also move this note: `Note that we have renamed DVC Studio to Iterative Studio and Views to Projects.` Coz the video uses the outdated terms.

Reply:
Updated.

Reply:
Sounds good. We should always use `<admon>` though, I think (updating).